src/lib/components/profile/image-input/ImageInput.svelte

```diff
@@ -0,0 +1,63 @@
+<script lang="ts">
+  import deleteIcon from '$lib/icons/delete.svg'
+  import uploadIcon from '$lib/icons/upload.svg'
+  import { slide } from 'svelte/transition'
+
+  export let title: string
+  export let subTitle: string
+  export let handleChange: (event: Event) => void = () => {}
+  export let handleDelete: () => void = () => {}
+  export let isDeletable: boolean = true
+
+  export let files: FileList | null = null
+  export let imageUrl: string = ''
+  export let image: File | null = null
+
+  let input: HTMLInputElement
+
+  function deleteImage() {
+    image = null
+    imageUrl = ''
+    files = null
+    input.value = ''
+    handleDelete()
+  }
+
+  $: if (files) {
+    image = files[0]
+    imageUrl = URL.createObjectURL(files[0])
+  }
+</script>
+
+<label
+  in:slide
+  out:slide
```

Comment:
```suggestion
  transition:slide
```
https://learn.svelte.dev/tutorial/transition